packages/clients/meldemichel/src/plugins/AfmButton/AfmButton.vue

```diff
@@ -1,5 +1,9 @@
 <template>
-  <v-tooltip :left="!hasSmallWidth" :top="hasSmallWidth">
+  <v-tooltip
+    :left="!hasSmallWidth"
+    :top="hasSmallWidth"
+    :disabled="hasSmallDisplay"
+  >
```

Comment:
```suggestion
  <v-tooltip :left="!hasSmallWidth" :disabled="hasSmallDisplay">
```
`top` and `disabled` are true in the same situation, right? In that case, top could be omitted. Or is there something configurable (planned) so that this should stay in?

Reply:
`hasSmallWidth` and `hasSmallDisplay` are not the same variables:
- `hasSmallWidth` if the width of the map is smaller than `SMALL_DISPLAY_WIDTH`
- `hasSmallDisplay` is true, if either the height or the widht of the window` is smaller than `SMALL_DISPLAY_WIDTH`

Reply:
Oh, I forgot about map width and client width. Thanks for clarification. 🎩